### ISSUE TITLE
please help me.. strange thing

### ISSUE THREAD

**Issue by Wreeper** [ConnorLinfoot/CratesPlus#597]
Config.yml: https://pastebin.com/raw/dsrfBRWX
data.yml:
Data Version: 2
Crate Locations:
  test: []
  vote:
  - world|4000015|83|3999961
  ranks:
  - world|4000018|83|3999956
CraftBukkit version: git-Spigot-21fe707-e1ebe52 (MC: 1.8.8) (API version 1.8.8-R0.1-SNAPSHOT
 Plugins (30): Itemizer, WorldEdit, CombatLog, Essentials, ClearChat, GroupManager, NametagEdit, HolographicDisplays, WorldGuard, SkinsRestorer, SignEdit, EssentialsChat, KitCreator, Vault, CCLogger, OreGeneratorRevolution, MoneyNote, CratesPlus, Jobs, LWC, PreciousStones, SilkSpawners, Citizens, Shopkeepers, ChestCommands, RandomTeleport, CommandNPC, ChestShop, AuthMe, LiteBans

No error/stack trace.
I do not know how to reproduce.

So basically when i'm parsing the YML it gives no errors, what is the problem? The problem is that the Ranks Crate when i reload the plugin removes all of the winnings without even give an error.

What could it be?
